HiLo.java

```diff
@@ -0,0 +1,65 @@
+import java.util.Scanner;
+
+public class HiLo {
+   public static void main(String[] args) {
+      Scanner sc = new Scanner(System.in);
+      
+      System.out.println("Välkommen till HiLo!");
```

Comment:
Did you consider replacing several `System.out.println()` with only one by appending `\n` at the end of each sentence?